diff --git a/services/network/public/cpp/BUILD.gn b/services/network/public/cpp/BUILD.gn
index 634827fc6f36b3d424a738b6d1994ea8070f2c3b..bc03611e5fdcc2c0314d73da6c6cbe73374e5819 100644
--- a/services/network/public/cpp/BUILD.gn
+++ b/services/network/public/cpp/BUILD.gn
@@ -189,6 +189,7 @@ component("cpp") {
     "//services/proxy_resolver/public/mojom",
   ]
 
+  public_deps += [ "//brave/services/network/public/cpp" ]
   defines = [ "IS_NETWORK_CPP_IMPL" ]
 }
 
